### PR TITLE
feat - Make project with Hello World by Marlon

### PR DESCRIPTION
**Pull request by MarlonPrado**
Hello, this is my first pull request after quite some time. I created the project with the classic "Hello World" from me in Kotlin. I also added the respective readme.md document at the beginning and a .gitignore file to ignore those cache files generated during the compilation process😎✔

### CODE REVIEW

File: .gitignore.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Built application files
```
Comment (jorlugaqui) [inline .gitignore.txt]: We don't need this file. Please remove it.

File: app/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (jorlugaqui) [inline app/.gitignore]: This file should be located at the root folder.

**Review by jorlugaqui — CHANGES_REQUESTED** [review]
Please fix the gitignore file and location.

Final score: 4.0

**Review by MarlonPrado — COMMENTED** [review]
changues are applied

**Review by jorlugaqui — APPROVED** [review]
Chicos, apruebo el PR. Pero por favor, agreguen el .gitignore en la raíz del proyecto (no vi ninguno).

**Pull request merged**